integration_tests/dbt_project.yml

```diff
@@ -4,18 +4,20 @@ config-version: 2
 
 profile: 'integration_tests'
 
-models: 
-  sage_intacct:
-    +materialized: table
-    intermediate:
-      +materialized: table
+## Why are we defining the models config here? I am pretty sure we can remove
+models:  
+  sage_intacct: 
+    +materialized: table 
+    intermediate:  
+      +materialized: table 
 
-  sage_intacct_source:
-    +materialized: table
-    tmp:
-      +materialized: view
+  sage_intacct_source: 
+    +materialized: table 
+    tmp: 
+      +materialized: view 
 
 vars:
+  ## You actually do not need to define the sage_intact models here. You can remove and just keep the source
```

Comment:
The source declaration is the only one that is actually needed in the integration tests folder